service/payment.go

```diff
@@ -73,6 +80,11 @@ func (service *PaymentService) CreatePaymentSession(w http.ResponseWriter, req *
 	paymentResource.Reference = incomingPaymentResourceRequest.Reference
 	paymentResource.ID = generateID()
 
+	journeyURL := cfg.PaymentServiceURL + paymentResource.ID + cfg.PaymentServicePath
```

Comment:
what is the end result of this expected to be?

looks like it'll end up something like `https://payments.companieshouse.gov.uk/000000001/pay` ?

1) we'd need a path prefix (maybe there is one expected in the PaymentServiceURL from env vars but that could be missed/wrong and isn't how we usually manage URLs)
2) looks like this assumes the PaymentServiceURL will have the trailing slash
3) looks like this assumes the PaymentServicePath will have the leading slash